sdk/src/client/api/block_builder/input_selection/utxo_chains.rs

```diff
@@ -203,8 +187,8 @@ impl<'a> ClientBlockBuilder<'a> {
             };
 
             utxo_chain_inputs.push(InputSigningData {
-                output: Output::try_from_dto(&output_response.output, token_supply)?,
-                output_metadata: OutputMetadata::try_from(&output_response.metadata)?,
+                output: output_with_meta.output().to_owned(),
```

Comment:
We should make the fields on `OutputWithMetadata` `pub(crate)` so that we can destructure it instead of cloning here.

Reply:
This is actually a good point, Fixing it